pkg/logout/service.go

```diff
@@ -0,0 +1,57 @@
+package logout
+
+import (
+	"fmt"
+	"github.com/TurnsCoffeeIntoScripts/jira-api-resource/pkg/configuration"
+	"github.com/TurnsCoffeeIntoScripts/jira-api-resource/pkg/http/rest"
+	"github.com/TurnsCoffeeIntoScripts/jira-api-resource/pkg/service"
+	"net/http"
+	"regexp"
+	"strings"
+)
+
+type ServiceLogoutSession struct {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported type ServiceLogoutSession should have comment or be unexported](https://app.codacy.com/app/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=4570750)